test/jdk/java/awt/print/PrinterJob/PageDlgPrnButton.java

```diff
@@ -38,18 +38,20 @@
  * @build PassFailJFrame
  * @run main/manual PageDlgPrnButton
  */
-public class PageDlgPrnButton implements Printable
-{
+public class PageDlgPrnButton implements Printable {
     private static final String INSTRUCTIONS =
-            "You must have at least 2 printers available to perform this test.\n" +
             "This test brings up a native Windows page dialog.\n" +
             "Click on the Printer... button and change the selected printer. \n" +
             "Test passes if the printout comes from the new selected printer.";
 
     public static void main(String[] args) throws Exception {
-        if (PrinterJob.lookupPrintServices().length < 2) {
+        final int serviceCount = PrinterJob.lookupPrintServices().length;
+        if (serviceCount == 0) {
             throw new RuntimeException("Printer not configured or available.");
         }
+        if (serviceCount < 2) {
+            throw new SkippedException("The test requires at least 2 printers.");
+        }
```

Comment:
This doesn't compile now. You didn't add the import for `jtreg.SkippedException`, nor did you modify the jtreg test to compile `SkippedException`.